doc/classes/Camera2D.xml

```diff
@@ -115,6 +115,9 @@
 		<member name="custom_viewport" type="Node" setter="set_custom_viewport" getter="get_custom_viewport">
 			The custom [Viewport] node attached to the [Camera2D]. If [code]null[/code] or not a [Viewport], uses the default viewport instead.
 		</member>
+		<member name="delay" type="bool" setter="set_delay_enabled" getter="get_delay_enabled" default="true">
+			If [code]delay[/code] is set, there may be a frame delay between setting the camera position and the view changing.
```

Comment:
Isn't it a bit weird to have a property which is on by default but whose name and description make it clear that it's likely not a wanted behavior?

I get that we want to avoid regressions at this stage, but maybe it should be made more explicit and be a property that fixes the issue when enabled? I.e. `delay_fix` defaulting to `false`, or something that better describes what it does technically (`sync_to_scene_tree`? not so clear maybe)?

Reply:
Originally I did have it the other way around with the name `defer_scroll`. But I wasn't sure that name would make much sense to users.

When reduz looked over it he commented that he thought we might end up not even having the option for delay, and I agree in the long term but just for introduction and compatibility I think we need both, as some gameplay might be depending on the delay, which could be regarded as a 'feature'. :grin:

I'll change it to `delay_fix` with reverse logic.

Reply:
Changed to `delay_fix`.